docs/iot/guides/things-api-guide-5.0.md

```diff
@@ -0,0 +1,443 @@
+# API Usage
+
+SmartThings Thing API is a service that provides connection to SmartThings Cloud. Also, it acts as an interface for resource applications to be controlled by the SmartThings applications.
+
+**Figure: SmartThings Thing overall architecture**
```

Comment:
To maintain consistency, with all other Tizen docs, please give the figure caption below the figure.

Reply:
Ok. Done.